client/modules/accounts/templates/dropdown/dropdown.html

```diff
@@ -45,6 +50,15 @@
         </a>
       </li>
     {{/each}}
+    <!-- Getting started dropdown -->
+    <li id="onboarding" class="dropdown-apps-icon onboarding">
+      <a id="dropdown-apps-onboarding" title="Get Started">
+        <i class="fa fa-info-circle"></i>
+        <span class="icon-text">
+          Get Started
```

Comment:
you don't need to break the span here since it has a short text. Breaking it just creates an unnecessary extra line.

Reply:
Ok. Will look at this one